NitroxServer/Communication/LiteNetLib/LiteNetLibServer.cs

```diff
@@ -73,6 +73,7 @@ private async Task PortForwardAsync(ushort port)
 
         public override void Stop()
         {
+            playerManager.SendPacketToAllPlayers(new ServerStopped());
```

Comment:
Maybe we can wait here for like half a second so everyone has received the ServerStopped packet.